app/views/kaui/invoices/show.html.erb

```diff
@@ -7,6 +7,9 @@
       <% unless @invoice.invoice_number.blank? %>
         <%= Kaui.customer_invoice_link.call(@invoice, self) %>
       <% end %>
+      <% unless @invoice.invoice_number.blank? %>
```

Comment:
You can simplify and remove this (one `unless` for both calls).

Reply:
Combined the both logic in same condition.